attribution-graphs-frontend/attribution_graph/init-cg-link-graph.js

```
@@ -0,0 +1,339 @@
+  var yNumTicks= visState.isHideLayer ? data.byStream.length : n_layers + 1
+  c.y = d3.scaleBand(d3.range(yNumTicks), [c.height, 0])
+
+  c.yAxis = d3.axisLeft(c.y)
+    .tickValues(d3.range(yNumTicks))
+    .tickFormat(i => {
+      if (i % 2) return
+
+      return i == n_layers ? 'Lgt' : i == 0 ? 'Emb' : 'L' + i
+      var label = data.byStream[i][0].layerLocationLabel
+      var layer = +label.replace('L', '')
+      return isFinite(layer) && layer % 2 ? '' : label
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)